public/components/common/hooks/use-es-search.ts

```diff
@@ -72,7 +72,7 @@ const useEsSearch = ({ preAppliedFilters = [], preAppliedAggs = {}, size = 10 })
         setIsLoading(false);
       }
     })();
-  }, [indexPattern, query, filters, page]);
+  }, [indexPattern, query, filters, page, preAppliedAggs.buckets.terms.order]);
```

Comment:
`preAppliedAggs` may not have a `buckets.terms.order` property as it is optional. It's probably best to set the dependency as simply `preAppliedAggs`